legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/platform/DefaultExecutionNodeContext.java

```diff
@@ -82,6 +82,10 @@ else if (clazz.equals(PureDate.class) && value instanceof EngineDate)
             {
                 return (T) ((EngineDate) value).transformToPureDate();
             }
+            else if (clazz.equals(Long.class) && value instanceof String)
+            {
+                return (T) Long.valueOf(Long.parseLong((String)value));
```

Comment:
Do you need the call to Long.parseLong here? There is a Long.valueOf method that takes in a string and internally calls Long.valueOf on Long.parseLong

Reply:
Noted